### ISSUE TITLE
Truncated screen in Draw Text

### ISSUE THREAD

**Issue by dstein766** [wsaunders1014/laptop-fix#5]
Drawing Tools --> Draw Text:  Note that only about 1/2 of the dialog box is used and user must scroll inside that 1/2 box to see all the options, including the Reset and Save buttons.

This happens BOTH with and without MythicUI enabled.

**Comment by wsaunders1014** [comment]
Are you using version 0.0.1?

**Comment by dstein766** [comment]
No.  I'm using 0.0.3.

------ Original Message ------

**Comment by dstein766** [comment]
Sorry, should have spec'ed the versions in the initial reports:

Foundry 0.6.6
DND5e 0.96
Laptop-fix 0.0.3
TidyUI - Game Settings 0.1.18
MythicUI 0.8.3

Essentially, these were the most recent versions of everything as of 
about 45 minutes ago..

------ Original Message ------

**Comment by wsaunders1014** [comment]
Can you take a screengrab?

**Comment by dstein766** [comment]
Drawing Tools --> Draw Text --> click on map to start the process leads 
to...(picture).  If you can't recreate then sounds like I'll need to be 
more aggressive re: checking for interaction with other modules.




------ Original Message ------

**Comment by dstein766** [comment]
Hmm...screencap wasn't preserved in my response.  Need to figure out how to attach the graphic image.  Looking now, but any pointers?

**Comment by dstein766** [comment]
![GtErt3oXI8](https://user-images.githubusercontent.com/24942951/91889689-135c4700-ec43-11ea-8f54-3a49df3be792.png)

**Comment by dstein766** [comment]
TL;DR:  Full recreation environment is laptop-fix + Furnace 2.2 w/Advanced Drawing Tools setting ENABLED

Oh, interesting.  Turned off everything except TidyUI and laptop-fix.  

Previously, Drawing Tools --> Draw Text --> click in drawing was IMMEDIATELY opening up that weird-looking dialog, which in retrospect is already wrong.  With everything off clicking in the drawing puts me into the expected mode where I can drag/size the text box onscreen and directly enter text onscreen.  So there's clearly another module interaction going on here that's causing the dialog to pop up at the wrong time.

...and the winner is...The Furnace v2.2, when using the "Advanced Drawing Tools" setting enabled.  This setting is the cause of text entry going straight into the dialog (which is designed behavior).  That dialog box is still mangled when Laptop-fix is also enabled, however.

**Comment by wsaunders1014** [comment]
yeah I was thinking it might be Furnace because I couldn't replicate it with Tidy or Mythic. I'll give it a look.

**Comment by wsaunders1014** [comment]
Try 0.0.5 i made some changes
